### ISSUE TITLE
3.10.0: FTBFS "‘floor’ is not a member of ‘std’"

### ISSUE THREAD

**Issue by onlyjob** [lizardfs/lizardfs#412]
~~~~
/mnt/tmpssd/src/lizardfs/11/lizardfs-v.3.10.0-rc/src/common/slice_read_planner.cc:131:4: error: ‘floor’ is not a member of ‘std’ 
    std::floor(bandwidth_overuse_ * slice_traits::requiredPartsToRecover(slice_type_)), 
    ^ 
/mnt/tmpssd/src/lizardfs/11/lizardfs-v.3.10.0-rc/src/common/slice_read_planner.cc: In member function ‘std::unique_ptr<ReadPlan> SliceReadPlanner::buildPlanFor(uint32_t, uint32_t)’: 
/mnt/tmpssd/src/lizardfs/11/lizardfs-v.3.10.0-rc/src/common/slice_read_planner.cc:190:4: error: ‘floor’ is not a member of ‘std’ 
    std::floor(bandwidth_overuse_ * 
    ^ 
~~~~

**Comment by onlyjob** [comment]
Thanks!

**Comment by onlyjob** [comment]
~~~~
/build/lizardfs-3.10.0+dfsg/src/common/slice_read_planner.cc: In member function 'int SliceReadPlanner::addExtraParts(SliceReadPlan*, int, int, int)': 
/build/lizardfs-3.10.0+dfsg/src/common/slice_read_planner.cc:131:4: error: 'floor' is not a member of 'std' 
    std::floor(bandwidth_overuse_ * slice_traits::requiredPartsToRecover(slice_type_)), 
    ^~~ 
/build/lizardfs-3.10.0+dfsg/src/common/slice_read_planner.cc: In member function 'std::unique_ptr<ReadPlan> SliceReadPlanner::buildPlanFor(uint32_t, uint32_t)': 
/build/lizardfs-3.10.0+dfsg/src/common/slice_read_planner.cc:190:4: error: 'floor' is not a member of 'std' 
    std::floor(bandwidth_overuse_ * 
    ^~~ 
~~~~

**Comment by rushmorem** [comment]
`3.10.0` builds fine for me after applying https://github.com/lizardfs/lizardfs/commit/26a27dedb7bee36f3ac5f48e22b977ea001d5903 only.

**Comment by onlyjob** [comment]
Someone just needs to make sure that 26a27de is included to next release...

**Comment by onlyjob** [comment]
Fixed in 3.10.2. Thanks.
